### ISSUE TITLE
Line Graph do not display X-axis labels when its populated with only one value

### ISSUE THREAD

**Issue by cshirole** [infor-design/enterprise-ng#365]
Please check the attached image.

Steps to Reproduce the Issue:
1) Unzip line.demo.ts into ./src/app/line
2) Check the line graph

Expected Behavior:
Line Graph should display X-axis values.

**Comment by cshirole** [comment]
Attaching Files..
[line.demo.ts.zip](https://github.com/infor-design/enterprise-ng/files/2875797/line.demo.ts.zip)
![linegraph_issue](https://user-images.githubusercontent.com/42473822/52954148-65b27b80-33af-11e9-901e-e6350d22f721.PNG)

**Comment by tmcconechy** [comment]
What if you added a zero data point? I'm also wondering about the data, is that realistic? What does it mean to have 3 different January datapoints with different names?

**Comment by cshirole** [comment]
Yes the data is realistic, we have Login, Logout and Timeout as the Legends and user can search for one day data on daily interval. We encountered this issue in one of our widgets. Example for the data returned for the widget is below.
public lineData = [
	{data: [{name: "02/26", value: 1}],name: "Logins",id: '1'}, 
	{data: [{name: 'Jan', value: 22}],name: 'Component B',id: '2'}, 
	{data: [{name: 'Jan', value: 32}],name: 'Component C', id: '3'}
	];

**Comment by cshirole** [comment]
Yes, the data is realistic, We encountered this error in one of our widget where user can select one days data and can view it on daily interval. Please see sample data below.
 [
	{data: [{name: "02/26", value: 1}],name: "Logins",id: '1'}, 
	{data: [{name: "02/26", value: 0}],name: 'Logouts',id: '2'}, 
	{data: [{name: "02/26", value: 1}],name: 'Timeouts', id: '3'}
	];

**Comment by cshirole** [comment]
![ae](https://user-images.githubusercontent.com/42473822/53403380-8d21cd80-39d9-11e9-9d52-3763840d9232.PNG)

Attaching widget screen shot.

**Comment by pwpatton** [comment]
@tmcconechy  is this an EP issue or a EP-NG issue?

**Comment by tmcconechy** [comment]
Yes this is an EP issue. Could you move it please @clepore ?
I can reproduce this by dropping that data in. Even tried to test with 

```
  var dataset = [{
    data: [{
        name: 'Jan',
        value: 12,
        depth: 4
    }],
    name: 'Component A'
  }, {
    data: [{
        name: 'Jan',
        value: 22
    }],
    name: 'Component B'
  }, {
    data: [{
        name: 'Jan',
        value: 32
    }],
    name: 'Component C'
  }];
```

For some reason if only one row of data the label isnt showing. As a workaround you might consider using a different type of chart if only one data point. Like maybe a bar chart.
